WcaOnRails/config/locales/ko.yml

```diff
@@ -238,7 +238,7 @@ ko:
     #original_hash: 52249b5
     datetime_placeholder: '년-월-일 시간 (YYYY-MM-DD HH:MM)'
     #original_hash: 7136e3b
-    time_format: 12h
+    time_format: 12기준
```

Comment:
This is causing the build to fail, because this has to be either `12h` or `24h`.

I'll reach out to Ilkyoo and ask him to re-submit his translation.